content/aws/iam/authorization.md

```diff
@@ -9,26 +9,33 @@ You can access AWS by Web console or AWS CLI.
 
 # Authorization using AWS CLI
 
-Then set up your AWS credentials in `~/.aws/credentials`. This should be shared by all AWS accounts in the Organization.
+1. Install `aws-vault` if you have not done so already using the directions [here](https://github.com/99designs/aws-vault#installing)
```

Comment:
I think we should maybe move this to `tools/aws-vault.md` and instead link to it from here.

Reply:
@osterman to clarify - were you referring to moving all of the steps over to `tools/aws-vault.md` or just step `1`

Reply:
Nevermind there is a ton of overlap - so I'm moving this all out to `tools/aws-vault.md` instead

Reply:
Yea, I think just move it. Also, keep in mind that the current documentation hierarchy is malleable. We're still figuring out how to best organize the information, so please speak up if you have any suggestions =)